src/test/regress/expected/multi_foreign_key.out

```diff
@@ -431,12 +431,11 @@ SELECT create_distributed_table('referencing_table', 'ref_id', 'hash');
 
 (1 row)
 
--- verify that we skip foreign key validation when propagation is turned off
--- not skipping validation would result in a distributed query, which emits debug messages
 BEGIN;
 SET LOCAL citus.enable_ddl_propagation TO off;
```

Comment:
This test doesn't seem to test what it should anymore, based on the DEBUG message below. I think it needs to be changed like this.
```suggestion
SET LOCAL citus.skip_constraint_validation TO on
```

Also the comment that was removed above the transaction can probably be put back and changed a bit.